applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -1942,6 +1944,43 @@ void TwoFluidNavierStokes<TElementData>::load(Serializer &rSerializer)
     KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, BaseType);
 }
 
+
+template <class TElementData>
+void TwoFluidNavierStokes<TElementData>::GetValueOnIntegrationPoints(   const Variable<double> &rVariable,
+                                                                        std::vector<double> &rValues,
+                                                                        const ProcessInfo &rCurrentProcessInfo )
+{
+    if (rVariable == DIVERGENCE){
+
+        auto p_geom = this->pGetGeometry();
```

Comment:
A suggestion: since you are always using methods in Geometry (you are always using the `->` operator), I would recommend fetching a reference to the geometry directly (`auto& r_geom = this->GetGeometry`)
```suggestion
        auto& r_geom = this->GetGeometry();
```